### ISSUE TITLE
Waterfox 56.2.13 won't accept third party cookies

### ISSUE THREAD

**Issue by Telmesomething** [BrowserWorks/waterfox#1115]
**Describe the bug**
Waterfox 56.2.13 won't accept third party cookies. This wasn't a problem with 56.2.12. 

**To Reproduce**
1. https://www.zalando.nl/
2. Click on "Ik ga akkoord" to accept all cookies. 
3. Open prefs and click on Privacy --> Show cookies. Only the zalando site cookie shows. 
4. Without third party cookies the site blocks login which means users can't place orders there. 

**Expected behavior**
Waterfox should accept third party cookies when configured to do so. 

**Screenshots**
![privacy_new_profile](https://user-images.githubusercontent.com/33827636/63851836-3cdef480-c997-11e9-8fef-320c390e2501.PNG)

![new profile_zalando](https://user-images.githubusercontent.com/33827636/63851870-4ec09780-c997-11e9-84bf-0aff5f1b7668.PNG)

**Desktop (please complete the following information):**
 - Windows
 - 8.1

**Additional context**
Behaviour was tested in a brand new profile without any addons and only with the menu bar configured to show.

**Comment by pjpreilly** [comment]
.. try "Keep Until They Expire"

**Comment by Telmesomething** [comment]
I did but it didn't make any difference. 

However, I contacted Zalando support and they suggested creating a new password. After logging in with that third party cookies appeared as they should. 

Maybe the problem lies with the usual profile I use which has all manner of prefs configurations along with anti-tracking addons and somehow or other a new profile duplicates those settings in the background even though they're not visible in a default setup. 

Anyway, problem solved now.
